split.Console/Models/Deck.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace split.Console.Models
+{
+    public class Deck
+    {
+        Card[] Cards;
+        int CardsIndex;
+        const int CARDSINDECK = 52;
+
+        public Deck(int numberDecks)
+        {
+            Cards = new Card[numberDecks * CARDSINDECK];
+            CardsIndex = 0;
+
+            for (int i = 0; i < numberDecks; i++)
+            {
+                foreach (SuitType suit in Enum.GetValues(typeof(SuitType)))
+                {
+                    foreach (RankType rank in Enum.GetValues(typeof(RankType)))
+                    {
+                        Cards[CardsIndex++] = new Card(suit, rank);
+                    }
+                }
+            }
+            Shuffle();
```

Comment:
I wouldn't call the method in the constructor. You could pass the deck to shuffle, sure. I am not sure what the benefit of that would be, except maybe making it testing easier... But you want to know if your shuffling algorithm works. How would you test for that? #thereigowiththetesting